src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs

```diff
@@ -201,16 +202,21 @@ public void AddInlineStats(StringBuilder benchLogs)
                 "NodeStore.Hash: ".PadRight(LoggingConfiguration.ColumnLength - 2) +
                 this.crossChainTransferStore.TipHashAndHeight.HashBlock.ToString() + "  " +
                 "NodeStore.NextDepositHeight: ".PadRight(LoggingConfiguration.ColumnLength + 1) +
-                this.crossChainTransferStore.NextMatureDepositHeight.ToString().PadRight(8));
-        }
+                this.crossChainTransferStore.NextMatureDepositHeight.ToString().PadRight(8) + 
+                "NodeStore.HasSuspended: ".PadRight(LoggingConfiguration.ColumnLength + 1) +
+                this.crossChainTransferStore.HasSuspended().ToString().PadRight(8)
+                );
+       }
 
         public void AddComponentStats(StringBuilder benchLog)
         {
             benchLog.AppendLine();
             benchLog.AppendLine("====== Federation Wallet ======");
 
-            var items = this.federationWalletManager.GetSpendableTransactionsInWallet(1);
-            benchLog.AppendLine("Federation Wallet: ".PadRight(LoggingConfiguration.ColumnLength) + " Confirmed balance: " + new Money(items.Sum(s => s.Transaction.Amount)).ToString());
+            var balances = this.federationWalletManager.GetWallet().GetSpendableAmount();
+            benchLog.AppendLine("Federation Wallet: ".PadRight(LoggingConfiguration.ColumnLength) 
+                                + " Confirmed balance: " + balances.ConfirmedAmount.ToString().PadRight(LoggingConfiguration.ColumnLength) 
+                                + " Unconfirmed balance: " + balances.UnConfirmedAmount.ToString());
             benchLog.AppendLine();

```

Comment:
Could include an indication whether federation is active. Perhaps merge with PR 213 which includes it.

Reply:
yes we can merge that later on as things get into master 👍